### ISSUE TITLE
Open modals programatically

### ISSUE THREAD

**Issue by bauerj** [MarkRabey/material-modal#3]
Something like 

    Modal.open("#modal");
    Modal.close("#modal");

would be nice to have. In my application I wanted to open a modal when the user entered invalid data. Currently I just do something like

            var event = document.createEvent('Event');
            event.initEvent('click', false, false);
            document.getElementById('modal-button').dispatchEvent(event);

which is okay, I guess (at least it works :wink: ), but it would be awesome if your library had a more sane approach built-in.

btw: :+1: for your awesome work!

**Comment by MarkRabey** [comment]
Good call! A few other projects to clean up, then I'll take a look. Thanks for call my work "awesome" :)

**Comment by ob7** [comment]
Never happened eh?
